metabase/metabase
# Add inline collection name and description editing

Epic https://github.com/metabase/metabase/issues/22826
Demo https://www.loom.com/share/b5185421a1624a7b9e3a4221124f2292

How to test:
- Go to a collection that is not root or personal
- Make sure it's possible to edit it's title and description inline

<img width="882" alt="Screenshot 2022-06-24 at 16 04 21" src="https://user-images.githubusercontent.com/8542534/175541851-b9341c63-9ad5-498b-8869-89a82baa1a2a.png">
